geode-redis/src/integrationTest/java/org/apache/geode/redis/internal/executor/string/AbstractDecrByIntegrationTest.java

```diff
@@ -33,14 +34,15 @@
 public abstract class AbstractDecrByIntegrationTest implements RedisPortSupplier {
 
   private Jedis jedis;
-  private Random rand;
+  private Jedis jedis2;
```

Comment:
If we end up adding the additional test, could we also remove `jedis2`? it is not used.

Reply:
whoops-  thanks

Reply:
added test